apps/services/auth/ids-api/src/app/delegations/test/delegation-index.spec.ts

```diff
@@ -306,6 +306,98 @@ describe('DelegationsIndexService', () => {
           delegations.find((d) => d.fromNationalId === fromNationalId)?.validTo,
         ).toStrictEqual(updatedValidTo)
       })
+
+      it('should remove scopes from custom delegations index item', async () => {
+        // Act
+        await delegationIndexService.indexDelegations(user)
+
+        const fromNationalId = testcase.customDelegations[0].fromNationalId
+        const testDelegation = await delegationModel.findOne({
+          where: {
+            fromNationalId,
+            toNationalId: user.nationalId,
+          },
+        })
+
+        // Add remove scope from delegation
+        if (testDelegation) {
```

Comment:
Wondering why there is an if statement here and in line 366. Do you expect this to not exist at some point?

Reply:
Just to make typescript happy, `delegationModel.findOne` can return null 🤷‍♀️